authors/apps/articles/serializers.py

```diff
@@ -5,19 +5,60 @@
 from rest_framework.pagination import PageNumberPagination
 from collections import OrderedDict
 from rest_framework.response import Response
+from .exceptions import ArticleNotFound
+from authors.apps.profiles.serializers import UserProfileSerializer
 
 
-class ArticleSerializer(serializers.ModelSerializer):
+class ArticleSerializer(UserProfileSerializer):
```

Comment:
why is the inheritance from the `UserProfileSerializer`

Reply:
The I am reusing its __init__ method. So repeating the code on the articles serializers would be redundant hence wouldn't pass on code climate